pyproject.toml

```diff
@@ -11,6 +11,7 @@ dependencies = [
     "opentelemetry-api",
     "opentelemetry-sdk",
     "opentelemetry-exporter-otlp-proto-http",
+    "sphinx>=7.4.7",
```

Comment:
We should make this a dev dependency instead to avoid users having to install all the extra dependencies that come with sphinx. https://docs.astral.sh/uv/concepts/dependencies/#development-dependencies

Reply:
I agree. We can do this with this cmd

```zsh
rye add --dev sphinx
```